spec/forms/decidim/participatory_documents/suggestion_form_spec.rb

```diff
@@ -25,6 +25,18 @@ module ParticipatoryDocuments
         expect(form).to be_valid
       end
 
+      it "is valid without a file" do
+        form.file = nil
```

Comment:
You shouldn't do this, better to use the proper `let` constructions.
In this case you can create a context and use a `let(:example_file) { nil }` in it